R/studies.R

```diff
@@ -0,0 +1,10 @@
+#' Function to get studies of Bridge App
+#' 
+#' @return response from Bridge /studies endpoint
+get_studies <- function(){
+    tryCatch({
+        bridgeclient:::bridgeGET("/v5/studies") %>% .$items
+    }, error = function(e){
+        glue::glue("study not available in Bridge App")
```

Comment:
This error message doesn't make sense. This endpoint returns a list of the studies in this app, but if there were no studies in the app it would just return an empty list (`.$items`), rather than throw an exception. I think we should leave it up to Bridge to raise the exception since we are only making an API request in this function.

Reply:
changed to e$message return